lib/jekyll/site.rb

```diff
@@ -319,9 +323,9 @@ def cleanup
     # Returns nothing.
     def write
       each_site_file { |item|
-        item.write(dest) if item.regenerate?
+        item.write(dest) if regenerator.regenerate?(item)
       }
-      metadata.write unless full_rebuild?
+      regenerator.write unless full_rebuild?
```

Comment:
Alright, fixed. It was `metadata.write` before, which made more sense, but now it's renamed to `regenerator.write_metadata`.